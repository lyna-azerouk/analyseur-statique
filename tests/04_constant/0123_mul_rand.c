{
  int x;
  x = rand(-1,1) * rand (10,15);
  print(x);
}
