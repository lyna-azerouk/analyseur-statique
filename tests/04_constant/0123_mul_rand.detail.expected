-- DEBUG: const 0 ~> {0}
-- DEBUG: rand -1 1 ~> ⊤
-- DEBUG: rand 10 15 ~> ⊤
-- DEBUG: ⊤ * ⊤ ~> ⊤
-- DEBUG: is_bottom ⊤ ~> false
stat trace: tests/04_constant/0123_mul_rand.c:3.2-32: [ x in ⊤ ]
tests/04_constant/0123_mul_rand.c:4.2-11: [ x in ⊤ ]
stat trace: tests/04_constant/0123_mul_rand.c:4.2-11: [ x in ⊤ ]
stat trace: tests/04_constant/0123_mul_rand.c:1.0-5.1: [ ]
analysis ended
